async-nats/src/tls.rs

```diff
@@ -61,7 +61,15 @@ pub(crate) async fn load_key(path: PathBuf) -> io::Result<PrivateKey> {
     .await?
 }
 
-pub(crate) async fn config_tls(options: &ConnectOptions) -> io::Result<rustls::ClientConfig> {
+pub(crate) struct TLSOptions {
```

Comment:
Rust tends to not capitalize, e.g Url rather than URL.
 
```suggestion
pub(crate) struct TlsOptions {
```

Reply:
well, technically, that's not grammar-wise proper, but I prefer idiomatic code rather than grammar :).